<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NovoFornecedor</class>
 <widget class="QWidget" name="NovoFornecedor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novo Fornecedor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/mdpi.png</normaloff>../Icons/mdpi.png</iconset>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrição:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="tbNome">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalvar">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>680</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="tbDescricao">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>680</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="tbEndereco">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Cidade:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="tbCidade">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Endereço:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Bairro:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="tbBairro">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="tbNumero">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>801</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Telefone</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="btnTelefoneRemove">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>210</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblTelefone">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>381</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>541</width>
      <height>192</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>126</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>DDD</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Numero</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Referencia</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="tbTelefoneDDD">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTelefoneAdd">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="tbTelefoneNumero">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>DDD:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="tbTelefoneReferencia">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Referência:</string>
     </property>
    </widget>
   </widget>
   <zorder>tblTelefone</zorder>
   <zorder>frame</zorder>
   <zorder>btnTelefoneRemove</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>801</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Email</string>
   </property>
   <widget class="QPushButton" name="btnEmailRemove">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>210</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblEmail">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>381</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>541</width>
      <height>192</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>190</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Referencia</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="tbEmail">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEmailAdd">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="tbEmailReferencia">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Referência:</string>
     </property>
    </widget>
   </widget>
   <zorder>tblEmail</zorder>
   <zorder>frame_2</zorder>
   <zorder>tblEmail</zorder>
   <zorder>frame_2</zorder>
   <zorder>btnEmailRemove</zorder>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>label_8</zorder>
  <zorder>tbNome</zorder>
  <zorder>btnSalvar</zorder>
  <zorder>tbDescricao</zorder>
  <zorder>btnCancelar</zorder>
  <zorder>label_7</zorder>
  <zorder>tbEndereco</zorder>
  <zorder>label_9</zorder>
  <zorder>tbCidade</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>tbBairro</zorder>
  <zorder>label_12</zorder>
  <zorder>tbNumero</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
